freqtrade/optimize/backtesting.py

```diff
@@ -144,13 +144,15 @@ def __init__(self, config: Dict[str, Any]) -> None:
         self.init_backtest()
 
     def __del__(self):
+        logger.info('Inside Backtesting.__del__()')
         self.cleanup()
 
     @staticmethod
     def cleanup():
         LoggingMixin.show_output = True
         PairLocks.use_db = True
         Trade.use_db = True
+        logger.info('Cleaned Backtest')
```

Comment:
If you think this output is needed (I think it's not) - it should be debug level - not info.
otherwise it'll be additional (for the user pointless) output that will be visible after running backesting.